frontend/src/components/TemplateEditor/controls/__snapshots__/ControlPanelTextArea.test.js.snap

```diff
@@ -51,6 +51,7 @@ exports[`ControlPanelTextArea component renders as expected 1`] = `
             class="pf-c-form-control"
             data-testid="area-controlId"
             id="controlId"
+            placeholder="Enter creation.app.name"
```

Comment:
I am a bit concerned that `creation.app.name` is not translated here. 
- In general, I don't think we should be using the default placeholders that substitute translated strings into other translated strings. But this may be difficult to enforce since temptifly is not written in Typescript.
- Is the `name` property for controls of type `singleselect` translated?  It appears to be from here: https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataAWS.js#L696 but I don't see where that happens. But if it is translated, I would expect it to be translated everywhere it is used.

Reply:
What if we pull out name and translate before putting into the placeholder?
```

        // if placeholder missing, create one
        let { placeholder } = control
        const translatedName = i18n(name)
        if (!placeholder) {
            placeholder = i18n('creation.ocp.cluster.enter.value', [translatedName ? translatedName.toLowerCase() : ''])
        }
```

Reply:
On line 23 it is not translated either, but these must be translated in practice. So - perhaps there is preprocessing on the control data that runs certain fields through translation? (And which wouldn't be at play when you create these components directly for testing.) Can you please try to verify that? If so, then what is currently in this snapshot would be the expected behaviour I guess. 

@jeswanke may know...

Reply:
Here it is:
https://github.com/stolostron/console/blob/main/frontend/src/components/TemplateEditor/utils/initialize-control-data.js#L106